L1Trigger/L1CaloTrigger/python/test_L1CaloJets_cfg.py

```diff
@@ -0,0 +1,102 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Should this test file (together with test_L1EGCrystalClusterEmulator_cfg.py) be under `L1Trigger/L1CaloTrigger/test` instead?

Reply:
The test folder was not there when I took over the code. I would probably leave the files where they are in case moving them breaks something.

Reply:
I think the test code should be in the test directory anyways, except there is a reason to be here. Could you please move?